jobs/gcp-discovery/src/main/java/com/tmobile/pacbot/gcp/inventory/file/AssetFileGenerator.java

```diff
@@ -34,41 +34,41 @@ public class AssetFileGenerator {
 	 */
 	private static final Logger log = LoggerFactory.getLogger(AssetFileGenerator.class);
 
-	@Autowired
-	VMInventoryCollector vmInventoryCollector;
-	@Autowired
-	FirewallInventoryCollector firewallInventoryCollector;
-	@Autowired
-	StorageCollector storageInventoryCollector;
-	@Autowired
-	CloudSqlInventoryCollector cloudSqlInventoryCollector;
-
-	@Autowired
-	BigQueryInventoryCollector bigQueryInventoryCollector;
-
-	@Autowired
-	PubSubInventoryCollector pubSubInventoryCollector;
-
-	@Autowired
-	KmsKeyInventoryCollector kmsKeyInventoryCollector;
-
-	@Autowired
-	DataProcInventoryCollector dataProcInventoryCollector;
+//	@Autowired
+//	VMInventoryCollector vmInventoryCollector;
+//	@Autowired
```

Comment:
I think this commented code was left out post-testing. Can you please check?

Reply:
sure @santhosh-challa